frameworks/Java/voovan/src/main/java/org/voovan/VoovanTFB.java

```diff
@@ -11,40 +11,46 @@
 import org.voovan.tools.json.JSON;
 import org.voovan.tools.log.Logger;
 
+import java.util.Map;
+
 
 public class VoovanTFB {
-    private static final byte[] HELLO_WORLD = "Hello, World!".getBytes();
-
-    public static void main(String[] args) {
-        WebServerConfig webServerConfig = WebContext.getWebServerConfig();
-	webServerConfig.setGzip(false);
-	webServerConfig.setAccessLog(false);
-	webServerConfig.setKeepAliveTimeout(20000);
-	webServerConfig.setHost("0.0.0.0");
-	webServerConfig.setPort(8080);
-	webServerConfig.setCache(true);
-	webServerConfig.getModuleonfigs().clear();
-	webServerConfig.getRouterConfigs().clear();
-        WebServer webServer = WebServer.newInstance(webServerConfig);
-        Logger.setEnable(false);
-
-        //性能测试路由
-        webServer.get("/plaintext", new HttpRouter() {
-            public void process(HttpRequest req, HttpResponse resp) throws Exception {
-                resp.header().put(HttpStatic.CONTENT_TYPE_STRING, "text/plain");
-                resp.header().remove(HttpStatic.CONNECTION_STRING);
-                resp.write(HELLO_WORLD);
-            }
-        });
-
-        //性能测试路由
-        webServer.get("/json", new HttpRouter() {
-            public void process(HttpRequest req, HttpResponse resp) throws Exception {
-                resp.header().put("Content-Type", "application/json");
-                resp.write(JSON.toJSON(TObject.asMap("message", "Hello, World!")));
-            }
-        });
-
-        webServer.serve();
-    }
+	private static final byte[] HELLO_WORLD = "Hello, World!".getBytes();
+	private static final Map MAP = TObject.asMap("message", "Hello, World!");
```

Comment:
@nbrady-techempower @helyho 
This change is not OK. The JSON object must be instantiated for each request. This is per test rules. More details for example here: #2888

Reply:
Whoops! Thanks @zloster!